Consul/Client_Results.cs

```diff
@@ -44,6 +44,24 @@ public ConsulResult(ConsulResult other)
     /// </summary>
     public class QueryResult : ConsulResult
     {
+        public enum CacheResult
+        {
+            UNDEFINED,
+            MISS,
```

Comment:
I get IntelliSense warning on these name, I think `Miss` and `Hit` should be used.

Reply:
Hmm.. It's not appearing for me :/